godep/strip/strip.go

```diff
@@ -58,6 +58,9 @@ func stripGodepWorkspaceHandler(path string, info os.FileInfo, err error) error
 	if path == vPath {
 		return nil
 	}
+	if info == nil {
+		return nil
+	}
```

Comment:
I wonder if the issue is also that an unhandled error is being passed as an argument into the function. That info is null because there is an error.

We ran into a similar problem in the vendor stripping handler. You can see that [here](https://github.com/Masterminds/glide/blob/9ef6ca313b3200079861f2de4e567b1d22081d19/path/strip.go#L13-L18).

Could you look to see if handling an error being passed in fixes the problem like the other function (linked) does?

Reply:
Switching to your proposed solution did not solve the problem, the root cause is `vendor/github.com/mesos/mesos-go/api/v0/examples/Godeps/_workspace/.gitignore` which is being removed earlier in the process of invoking `glide update --strip-vendor` and then leads to following error:
```
[ERROR]	Unable to strip vendor directories: lstat /data/github.com/openshift/origin/vendor/github.com/mesos/mesos-go/api/v0/examples/Godeps/_workspace/.gitignore: no such file or directory`
```